checker/tests/interning/Raw3.java

```diff
@@ -54,7 +54,7 @@
   // with raw types
   List<@Interned String> foo6() {
     List<String> sl = new ArrayList<>();
-    //:: warning: [unchecked] unchecked conversion
+    //:: warning: [unchecked] unchecked conversion :: error: (return.type.incompatible)
```

Comment:
I tend to agree that if javac already issues a warning, the Checker Framework probably doesn't also need one.  (Except that we really want people not to use raw types when using the Checker Framework!)